playbooks/openshift-checks/pre-install.yml

```diff
@@ -0,0 +1,15 @@
+---
+- include: ../openshift-cluster/initialize_groups.yml
+
+- hosts: OSEv3
+  # Temporarily reverting to OSEv3 until group standardization is complete
+  name: run OpenShift pre-install checks
+  roles:
+    - openshift_health_checker
+  post_tasks:
+    # NOTE: we need to use the old "action: name" syntax until
+    # https://github.com/ansible/ansible/issues/20513 is fixed.
```

Comment:
Until we have something in CI that tests the playbooks, I think it is worth a quick link like:

    - action: openshift_health_check  # NOTE: https://github.com/ansible/ansible/issues/20513

Also reminds us to check the issue and "fix" the YAML when it's ready.